crates/matrix-sdk/src/sliding_sync/room.rs

```diff
@@ -41,12 +41,9 @@ impl SlidingSyncRoom {
     pub(super) fn new(
         client: Client,
         room_id: OwnedRoomId,
-        mut inner: v4::SlidingSyncRoom,
+        inner: v4::SlidingSyncRoom,
         timeline: Vec<SyncTimelineEvent>,
     ) -> Self {
-        // we overwrite to only keep one copy
-        inner.timeline = vec![];
```

Comment:
Why was this removed? It's not clear.

Reply:
Because it's not necessary anymore (I guess it wasn't necessary before, but now I'm sure).